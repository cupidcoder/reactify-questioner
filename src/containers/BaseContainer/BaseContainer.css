.BaseContainer {
  max-width: 1140px;
  margin: 3rem auto;
}
