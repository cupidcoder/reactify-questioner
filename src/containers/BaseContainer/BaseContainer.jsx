import React from 'react';
import style from './BaseContainer.css';

const BaseContainer = ({ children }) => (
  <div className={style.BaseContainer}>
    {children}
  </div>
);

export default BaseContainer;
