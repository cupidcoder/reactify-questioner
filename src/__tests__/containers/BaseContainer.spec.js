import React from 'react';
import { shallow } from 'enzyme';
import BaseContainer from '../../containers/BaseContainer/BaseContainer';

describe('BaseContainer', () => {
  it('Should correctly save snapshot of the BaseContainer', () => {
    const wrapper = shallow(<BaseContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
