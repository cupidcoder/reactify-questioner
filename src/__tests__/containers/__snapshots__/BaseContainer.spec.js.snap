// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseContainer Should correctly save snapshot of the BaseContainer 1`] = `ShallowWrapper {}`;
